db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Operations/Methods.class/__methods__/available_vdc_networks.rb

```diff
@@ -0,0 +1,46 @@
+class AvailableVdcNetworks
+  def initialize(handle = $evm)
+    @handle = handle
+  end
+
+  def main
+    fill_fialog_field(fetch_list_data)
+  end
+
+  def fetch_list_data
+    service = @handle.root.attributes["service_template"] || @handle.root.attributes["service"]
+    vdc_networks = service.try(:orchestration_manager).try(:cloud_networks).try(:select) do |net|
+      net.type == "ManageIQ::Providers::Vmware::NetworkManager::CloudNetwork::OrgVdcNet"
+    end
+    vdc_networks_list = vdc_networks.each_with_object({}) { |net, hash| hash[net.ems_ref] = net.name } if vdc_networks
```

Comment:
Replacing this line with

```
    vdc_networks_list = { vdc_networks[0].ems_ref => vdc_networks[0].name }
```

works... So, basically, I am just hardcoding the one network that I know I have in my inventory.